src/opensearch.py

```diff
@@ -109,11 +111,11 @@ def _build_paths(self) -> Paths:
           - OPENSEARCH_CONF: writeable by root or snap_daemon ($SNAP_COMMON) where config files are
         """
         return Paths(
-            home=f"{self._SNAP_DATA}",
-            conf=f"{self._SNAP_DATA}/config",
+            home=f"{self._ETC}",
```

Comment:
This does not seem to match what the snap has, [here](https://github.com/canonical/opensearch-snap/pull/35/files#diff-56759910381a014fecfd7556dd72ddd68c747d922a5b7df2044b9ce7c552f5f5R49) - is the snap structure wrong?

Reply:
nope, I got this wrong.